src/win32/tip/tip_text_service.cc

```diff
@@ -511,8 +510,6 @@ class TipTextServiceImpl
     StorePointerForCurrentThread(this);
 
     HRESULT result = E_UNEXPECTED;
-    RegisterLogFileSink(
-        FileUtil::JoinPath(SystemUtil::GetLoggingDirectory(), kLogFileName));
```

Comment:
I'm actually wondering if we can permanently deprecate the log file, as it's not reliable (we cannot write file from AppContainer process) and I guess no one (including me) is checking it as it is only available in debug build.

Are you still using `*.log` from other processes?  I'm actually leaning towards deprecating other logging files, but it's a separate discussion after fixing this obvious crash bug.

Reply:
I'm fine to permanently deprecate the log file.
I'd like to prevent anyone from creating the log file again in the future, since it may cause the crash issue again.

So adding a note instead of the comment out is also fine with me.